x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -385,7 +354,7 @@ export class VectorStyle extends AbstractStyle {
     );
   }
 
-  _getStyleFields() {
+  _getFeatureStateParams() {
```

Comment:
Not sure if `_getFeatureStateParams` is an accurate name. Some of the results returned from this function will not support feature state, like icon-color. Maybe a name not tied to mapbox feature-state, like `_getFeatureStyleParams`